packages/cli/src/commands/source/push/list.ts

```diff
@@ -0,0 +1,80 @@
+import {FilterHostType, SourceModel} from '@coveord/platform-client';
+import {cli} from 'cli-ux';
+import {Command} from '@oclif/command';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import dedent from 'ts-dedent';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../../hooks/analytics/analytics';
+
+export default class SourcePushList extends Command {
+  public static description =
+    'List all available push sources in your Coveo organization';
+
+  public static flags = {
+    ...cli.table.flags(),
+  };
+
+  @Preconditions(IsAuthenticated())
+  public async run() {
+    const {flags} = this.parse(SourcePushList);
+    const authenticatedClient = new AuthenticatedClient();
+    const org = (await authenticatedClient.cfg.get()).organization;
```

Comment:
Is it a styling preference that the `org` variable is declared at the beginning even if it is only used in the `if` block?
Really not important

Reply:
Yeah, styling preference I guess :D